lib/customer-deposit-wallet/src/Cardano/Wallet/Deposit/Pure/State/Creation.hs

```diff
@@ -114,27 +132,41 @@ fromCredentialsAndGenesis credentials knownCustomerCount genesisData =
     network = Read.getNetworkId genesisData
     initialUTxO = mempty
 
+mkMnemonic
+    :: Text -> Either (MkSomeMnemonicError '[15, 24]) SomeMnemonic
+mkMnemonic = mkSomeMnemonic . T.words
```

Comment:
Is it a convention to use `mkXX` when creating a `XX` with the possibility of failure? Perhaps a more descriptive name would clarify the intent, e.g `textToMnemonic`, or `mnemonicFromText` or `validateMnemonic`?
Also, I would add some property asserting the behaviour of this function, eg. that it accepts well formed 15 words or 24 words mnemonics. 
Perhaps this is overkill as we are relying on an existing (and hopefully tested) function from another library? But we have the `T.words` call, and what about newlines separated mnemonics?

Reply:
It's tested in DerivationSpec.hs

Reply:
Renamed